less/v2/apps.less

```
@@ -442,6 +442,38 @@
     font-size: var(--font-size-11);
   }
 
+  .separator.ampersand {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 4px;
+
+    &::before, &::after {
+      content: "";
+      width: 50%;
+      height: 1px;
+    }
+
+    &::before { background: linear-gradient(to left, var(--dnd5e-color-gold), transparent); }
+    &::after { background: linear-gradient(to right, var(--dnd5e-color-gold), transparent); }
+
+    > div {
+      width: 20px;
+      height: 20px;
+      background: url("ui/official/ampersand-gold.svg") no-repeat center / contain;
+    }
+  }
```

It might make sense to use an `<hr>` element so we can get the semantic separator in addition to the visual styling. We could even get rid of the second element by massaging the CSS a bit:

```html
<hr class="ampersand">
```

```suggestion
  hr.ampersand {
    position: relative;
    height: 20px;
    border: none;
    background: url("ui/official/ampersand-gold.svg") no-repeat center / contain;

    &::before, &::after {
      content: "";
      position: absolute;
      inset-block: 9.5px;
      inset-inline: 0;
    }
    &::before {
      background: linear-gradient(to left, var(--dnd5e-color-gold), transparent);
      inset-inline-end: calc(50% + 14px);
    }
    &::after {
      background: linear-gradient(to right, var(--dnd5e-color-gold), transparent);
      inset-inline-start: calc(50% + 14px);
    }
  }
```

---

Good idea